source/backend/dal/Helpers/Extensions/PersonExtensions.cs

```diff
@@ -0,0 +1,29 @@
+using System.Text;
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Helpers.Extensions
+{
+    public static class PersonExtensions
+    {
+        public static string FormatName(this PimsPerson pimsPerson)
```

Comment:
is this necessary give that GetFullName extension method already exists?

Reply:
the existing extension method doesn't include Suffix